Formula/kubernetes-helm@3.rb

```diff
@@ -0,0 +1,51 @@
+class KubernetesHelmAT3 < Formula
+  desc "The Kubernetes package manager"
+  homepage "https://helm.sh/"
+  url "https://github.com/helm/helm.git",
+      :tag      => "v3.0.0-beta.5",
+      :revision => "509a8819b7e8dba86b7982c006fce22628aeea28"
+  head "https://github.com/helm/helm.git"
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "f055d13d28b5f76e19718cb93bd5e64f34e27c1602de39fb6ade3449756153cc" => :catalina
+    sha256 "245bdf17ac134d98f7e022ddd36678ae62c0aa02116d3a61229f5fc1c5de5665" => :mojave
+    sha256 "859c2dd730bf2e6c593ca19bbc18a9e155470193770a7824ff1ad74f906ed7b9" => :high_sierra
+  end
+
+  depends_on "glide" => :build
```

Comment:
(For when Helm 3 is eventually released)

Helm 3 doesn't rely on Glide compared to Helm 2 - we migrated to Go Modules.